scss/components/_latest.scss

```diff
@@ -0,0 +1,116 @@
+@import "headline";
+@import "folder";
+@import "./mixins/flex-mix.scss";
+@import "./src/fonts";
+@import "./blog";
+
+.latest-portfolio {
+  background: $white;
```

Comment:
Do not use color names for variable names.

Reply:
Done